libs/use-viewer/src/index.ts

```diff
@@ -117,6 +118,11 @@ const useViewer = ({ handleError, streamAccountId, streamName, subscriberToken }
             handleInternalError(error);
           }
         }
+
+        if (!remoteTrackSourcesRef.current?.size) {
```

Comment:
Is it working when there are 4 sources, 2 of them are simulcast while the other 2 are not?

Reply:
When there are multiple sources that have different simulcast settings, the "layer" event triggers when we switch between main source. This automatically resets the quality options to "auto" only

Reply:
Will layer event be triggered when switch from simulcast source to non-simulcast source?

Reply:
Yes, I tested it using Vishal's example (second checkbox in the issue description) and verified the layer event was triggering